apps/dashboard/src/components/molecules/FilterableTable/FilterableTable.tsx

```diff
@@ -39,8 +39,8 @@ export function FilterableTable<T>({ tableOptions, filters }: FilterableTablePro
   })
 
   return (
-    <Card>
-      <Group mb="md">
+    <Card p={0}>
```

Comment:
```suggestion
    <Card p={0} bg="inherit">
```
To remove bg in darkmode now there's no padding